### PR TITLE
enhance(macros/EmbedYouTube): add optional title attribute

### PR DESCRIPTION
**Pull request by bsmth**
__edit:__ reverting to change title only per reviewer feedback

## Summary

This PR adds a few more parameters to the YouTube embed macro.

### Problem

Currently the title, width and height are hardcoded, so it would be nice to be able to customize these.

### Solution

Add three optional parameters that fall back to current hardcoded values if not given.

## Screenshots

image

## How did you test this change?

```md
{{EmbedYouTube("hs3piaN4b5I", "Awesome Flexbox video", 200, 100)}}
```

```
yarn && yarn dev
```

### CODE REVIEW

File: kumascript/macros/EmbedYouTube.ejs [hunk 1]
```diff
@@ -3,12 +3,17 @@
 //
 // Parameters:
 //  $0  Video ID
+//  $1  title (optional)
+//  $2  width (optional)
+//  $3  height (optional)
```
Comment (caugner) [inline kumascript/macros/EmbedYouTube.ejs:8]:
```suggestion
```

File: kumascript/macros/EmbedYouTube.ejs [hunk 2]
```diff
@@ -3,12 +3,17 @@
...
 
-var video = '<iframe width="560" height="315" ' +
+var video = '<iframe ' +
   `src="https://www.youtube-nocookie.com/embed/${$0}" ` +
+  `title="${$1 || 'YouTube video'}" ` +
+  `width="${$2 || 560}" ` +
+  `height="${$3 || 315}" ` +
```
Comment (caugner) [inline kumascript/macros/EmbedYouTube.ejs:14]:
```suggestion
var video = '<iframe width="560" height="315" ' +
  `src="https://www.youtube-nocookie.com/embed/${$0}" ` +
  `title="${$1 || 'YouTube video'}" ` +
```

**Review by caugner — CHANGES_REQUESTED** [review]
Let's only add the title now, for accessibility, as the width/height is a separate issue.

**Review by caugner — APPROVED** [review]

**Comment by caugner** [comment]
FYI @fiji-flo

**Comment by bsmth** [comment]
Thank you!

**Pull request merged**